Formula/tensorflow.rb

```diff
@@ -0,0 +1,52 @@
+class Tensorflow < Formula
+  desc "Machine Intelligence Library"
+  homepage "https://tensorflow.org"
+  url "https://github.com/tensorflow/tensorflow.git",
+    :tag => "v0.7.1",
+    :revision => "028d0b46004c921acd48fdd0ec18128d79e18bf4",
+    :shallow => false
+
+  env :std
```

Comment:
We won't accept a new formula using `env :std`. Please explain why you need it, Thanks.